components/Header.tsx

```diff
@@ -3,7 +3,10 @@ import Link from 'next/link';
 
 export default function Header({ coordinatorName }) {
   return (
+    <>
+    <title> {`Vortex | ${coordinatorName}`} </title>
```

Comment:
To make sure this goes in the `head` of the page you can `import Head from 'next/head'` and wrap it in a `Head` component.

Reply:
Done